### PR TITLE
Added default future date for activation time

### PR DESCRIPTION
**Pull request by roienatan**
When creating a proposal, activation time is now given a default date for the same hour the next day.

### CODE REVIEW

File: src/components/Proposal/Create/PluginForms/CreatePluginManagerProposal.tsx [hunk 1]
```diff
@@ -162,7 +163,7 @@ class CreatePluginManagerProposal extends React.Component<IProps, IState> {
       votersReputationLossRatio: 4,
       minimumDaoBounty: 150,
       daoBountyConst: 10,
-      activationTime: new Date(),
+      activationTime: moment().add(1, "day").format("YYYY-MM-DDTHH:mm"), //default date for the next day
```
Comment (orenyodfat) [inline src/components/Proposal/Create/PluginForms/CreatePluginManagerProposal.tsx:166]: how about the default date value will be 0 (1.1.1970 00:00) ?
in that case there will be no constraint re the activation time by default .at the contract level .

Comment (roienatan) [reply src/components/Proposal/Create/PluginForms/CreatePluginManagerProposal.tsx:166]: Tha nature of activation time is to be a future date, so I think it'll be confusing if the default date will be a past date.

Comment (roienatan) [reply src/components/Proposal/Create/PluginForms/CreatePluginManagerProposal.tsx:166]: Also from a UX perspective, as the rest of the default params that are initialized are all valid, so this one also must be valid.

Comment (orenyodfat) [reply src/components/Proposal/Create/PluginForms/CreatePluginManagerProposal.tsx:166]: ok. lets have it this way for now. 👍

**Pull request merged**